### DISCUSSION TITLE
cmake error on macos

### DISCUSSION THREAD

**Discussion by Tony-qty** [google/filament#5499] [Q&A]
![image](https://user-images.githubusercontent.com/7675144/166193366-570d773e-4fa7-4edc-b7ea-15903ff7c16c.png)

etc

**Comment by bejado** [comment]
We don't support Xcode as a build system generator and recommend using Ninja instead. That being said, you may want to try experimenting with [`CMAKE_XCODE_BUILD_SYSTEM`](https://cmake.org/cmake/help/latest/variable/CMAKE_XCODE_BUILD_SYSTEM.html).
